samples/core/blur/main.cpp

```diff
@@ -85,22 +87,23 @@ int main(int argc, char* argv[])
             // file you can find 'USE_SUBGROUP_EXCHANGE_RELATIVE' C-like
             // definition switch for blur_box_horizontal_subgroup_exchange
             // function. In this case, 2 blur kernel functors are used.
-            if (use_subgroup_exchange_relative)
+            if (use_subgroup_exchange_relative && opencl_version_2_0)
             {
                 std::cout << "Dual-pass subgroup relative exchange blur"
                           << std::endl;
-
-                blur.build_program("-D USE_SUBGROUP_EXCHANGE_RELATIVE ");
+                // cl_khr_subgroup_shuffle_relative requires OpenCL 2.0
+                blur.build_program(
+                    "-D USE_SUBGROUP_EXCHANGE_RELATIVE -cl-std=CL2.0 ");
                 blur.dual_pass_subgroup_exchange_box_blur();
             }
 
             // Same as the previous one, but with a different build switch. See
             // the blur.cl file for more info about the switch.
-            if (use_subgroup_exchange)
+            if (use_subgroup_exchange && opencl_version_2_0)
             {
                 std::cout << "Dual-pass subgroup exchange blur" << std::endl;
-
-                blur.build_program("-D USE_SUBGROUP_EXCHANGE ");
+                // cl_khr_subgroup_shuffle requires OpenCL 2.0
+                blur.build_program("-D USE_SUBGROUP_EXCHANGE -cl-std=CL2.0 ");
```

Comment:
See comment above: there is no dependency between `cl_khr_subgroup_shuffle` and OpenCL C 2.0.